src/main/java/com/WebController/EventController.java

```diff
@@ -63,4 +81,53 @@ public String receivePatient (@RequestBody List<Event> events){
         return list;
     }
 
+    @PostMapping(value = "/userregister", consumes = "application/json")
+    @ResponseBody
+    public String userRegistration(@RequestBody User user){
+        if (this.userRepository.findUserByEmailAddressIgnoreCase(user.getEmailAddress())!=null){
+            return "user already exists";
+        }
+        userRepository.save(user);
+        ConfirmationToken token = new ConfirmationToken(user);
+
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setTo(user.getEmailAddress());
+        message.setSubject("COVID-19 TRacking System Registration.");
+        message.setFrom("covidtracking08@gmail.com");
+        message.setText("To confirm your email address, please click here:"
+        + "http://localhost:8080/confirmaccount?token="+token.getConfirmationToken());
+        emailSenderService.sendEmail(message);
+        this.confirmationTokenRepository.save(token);
+        return "Confirmation email sent to: "+user.getEmailAddress();
+    }
+
+    @RequestMapping (value = "/confirmaccount", method = {RequestMethod.POST, RequestMethod.GET})
+    public String ConfirmEmail (@RequestParam("token") String confirmationToken, Model model){
+        ConfirmationToken token = this.confirmationTokenRepository.findByConfirmationToken(confirmationToken);
+        if (token!=null){//valid user
+            String emailAddress = token.getUser().getEmailAddress();
+            User user = userRepository.findUserByEmailAddressIgnoreCase(emailAddress);
+            user.setEnable(true);
+            userRepository.save(user);
+            model.addAttribute("emailAddress", emailAddress);
+            return "RegisterSuccess";
+        }else {
+            return "Error";
+        }
+    }
+
+    @PostMapping (value = "/userlogin", consumes = "application/json")
+    @ResponseBody
+    public String login (@RequestBody User longinUser){
+        User user = this.userRepository.findUserByEmailAddressIgnoreCase(longinUser.getEmailAddress());
+        if (user!=null){
+            if (user.isEnabled()){
+                return "success";
```

Comment:
I don't think a REST API should communicate with the client using Strings (the consumer of this API may not be a human). Failure here could return some HTTP code. Let the client decide how to display things if need be.